src/compiler/emitter.ts

```diff
@@ -6891,12 +6891,13 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 if (currentSourceFile.text.charCodeAt(comment.pos + 1) === CharacterCodes.asterisk) {

```

Comment:
i don't like this.  The function is called isPinnedOrTripleSlashComment.  And you're returning false for a triple slash comment.  I might call it instead "shouldEmitComment" instead.